client/src/components/CoinsCard.js

```diff
@@ -12,51 +13,53 @@ export default function Coinscard({ topCoins, onToggleFavoriteCoins }) {
 
   return (
     <>
-      {topCoins.map((topCoin, index) => (
-        <CoinWrapper key={index + topCoin}>
-          <section>
-            <CoinNameSymbole>
-              <CoinImage src={topCoin.image} alt="Image of Coin" />
-              <span>
-                {topCoin.name}{' '}
-                <FavoriteIcon
-                  src={favoriteIcon}
-                  alt="Star image of icon"
-                  onClick={() => onToggleFavoriteCoins(topCoin)}
-                  style={
-                    topCoin.isFavorite
-                      ? { opacity: '100%' }
-                      : { opacity: '30%' }
-                  }
-                />
-              </span>
-              <CoinSymbole>{topCoin.symbol}</CoinSymbole>
-            </CoinNameSymbole>
-            <CoinPriceChange>
-              <CoinPricePercent>Price / 24h % </CoinPricePercent>
-              <span>{topCoin.current_price.toFixed(2)} $</span>
-              {topCoin.price_change_percentage_24h > 0 ? (
-                <PositivePrice>
-                  {topCoin.price_change_percentage_24h.toFixed(2)} %
-                </PositivePrice>
-              ) : (
-                <NegativePrice>
-                  {topCoin.price_change_percentage_24h.toFixed(2)} %
-                </NegativePrice>
-              )}
-            </CoinPriceChange>
-            <CoinCapVolume>
-              <CoinMarketCapVolume>Market Cap/Volume</CoinMarketCapVolume>
-              <span>{ShortenLargeNumbers(topCoin.market_cap)}</span>
-              <span>{ShortenLargeNumbers(topCoin.total_volume)}</span>
-            </CoinCapVolume>
-          </section>
-        </CoinWrapper>
-      ))}
+      <CoinWrapper key={index + topCoin}>
+        <section>
+          <CoinNameSymbole>
+            <CoinImage src={topCoin.image} alt="Image of Coin" />
+            <span>
+              {topCoin.name}{' '}
+              <FavoriteIcon
+                src={favoriteIcon}
+                alt="Star image of icon"
+                onClick={() => onToggleFavoriteCoins(topCoin)}
+                style={
+                  topCoin.isFavorite ? { opacity: '100%' } : { opacity: '30%' }
```

Comment:
maybe you remove this inline type of styling and move it to the styled components section? :)